src/main/java/ca/wolfram/beta/symath/operations/SubtractNode.java

```diff
@@ -0,0 +1,42 @@
+package ca.wolfram.beta.symath.operations;
+
+import ca.wolfram.beta.symath.MathNode;
+import ca.wolfram.beta.symath.MathUtils;
+import ca.wolfram.beta.symath.NodeType;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by Allan Wang on 2017-05-11.
+ */
+public class SubtractNode {
+
+    private SubtractNode() {
+    }
+
+    /**
+     * Links all children with subtract operations
+     * If first node is 0, automatically creates a NegateNode with rest of contents
+     *
+     * @param nodes list of children
+     * @return resulting MathNode
+     * @throws IllegalArgumentException if nodes count < 2
+     */
+    public static MathNode create(List<MathNode> nodes) {
+        NodeType.SUBTRACT.validateListSize(nodes);
+        List<MathNode> toSubtract = nodes.subList(1, nodes.size());
+        MathNode subtrahend;
```

Comment:
It's like dividend divisor -> minuend subtrahend. Do you still want me to rename it?